website/src/controllers/groups.js

```diff
@@ -418,7 +436,10 @@ api.join = function(req, res, next) {
 
   if (!_.contains(group.members, user._id)){
     group.members.push(user._id);
-    group.invites.splice(_.indexOf(group.invites, user._id), 1);
+    //Can we have someone joing without an invite? I added this for testing since we join users without an invite
+    if (group.invites.length > 0) {
```

Comment:
@paglias Do you have context on this? I'd hate to break something by merging this in.